includes/admin/class-wc-admin-menus.php

```diff
@@ -52,15 +52,20 @@ public function __construct() {
 	 */
 	public function admin_menu() {
 		global $menu;
-
-		$icon_svg = file_get_contents( WC()->plugin_url() . '/assets/images/classic-commerce-dashicon-white-on-transparent.svg' );
-		$cc_icon = base64_encode( $icon_svg );
+		
+		$cc_icon = '';
+		$icon_file = WP_PLUGIN_DIR  . '/classic-commerce/assets/images/classic-commerce-dashicon-white-on-transparent.svg';
```

Comment:
There is a double space on this line after `WP_PLUGIN_DIR`.

My personal preference would also be to avoid hardcoding the plugin slug. It is rare but not completely unheard-of for people to rename plugins and modify them, and it seems worthwhile not to introduce any new issues related to that edge case.

Reply:
> It is rare but not completely unheard-of for people to rename plugins and modify them

I didn't intend this as a tongue-in-cheek statement when I wrote it, but that's exactly what we did with Woo → CC. Plugin forks and renames are another reason to avoid hardcoding the plugin slug if possible.

Reply:
Thought: Would a new constant like https://github.com/timbocode/classic-commerce/blob/02571f11556148977731f34ccd0323f1b0273622/classic-commerce.php#L148-L151
help solve the issue?

Use 

```
if( ! defined( 'CLASSICCOMMERCE_CORE_PLUGIN_URL' ) ) {
	define( 'CLASSICCOMMERCE_CORE_PLUGIN_URL', plugin_dir_url( __FILE__ ) );
}
```

This constant can always be used in several places.